app/src/main/kotlin/com/exallium/todoapp/MainActivity.kt

```diff
@@ -59,4 +66,12 @@ class MainActivity : AppCompatActivity() {
             super.onBackPressed()
         }
     }
+
+    override fun onOptionsItemSelected(item: MenuItem?): Boolean {
```

Comment:
we have to have onOptionsItemSelected() since we are not declaring parent activity in the manifest and we only have 1 activity in the project.